Check encumbrance helper function

skyleronken opened the pull request:
Noticed a few feature requests would require an encumbrance check. Added a helper function to DRC that will run encumbrace, capture the response message, and then return the numeric value from 0-11.

skyleronken commented:
,en echo DRC.check_encumbrance

--- Lich: exec1 active.

[exec1]>encumbrance

  Encumbrance : None (0/11)
> 
[exec1: 0]

--- Lich: exec1 has exited.

rpherbig commented on common.lic:
```diff
@@ -283,4 +298,11 @@ module DRC
+  
+  def check_encumbrance
+    enc = nil
+    enc_msg = bput('encumbrance', 'Encumbrance : .*')
```
You can get your current encumbrance (as a string) without sending a command via `DRStats.encumbrance`. You'll still have to map that, though.

rpherbig commented on common.lic:
```diff
@@ -283,4 +298,11 @@ module DRC
   end
+  
+  def check_encumbrance
+    enc = nil
```
Where is `enc` used?

skyleronken commented:
so I changed it to use DRStats.encumbrance. But DRStats is only updated when the encumbrance command is issued. So I added a default true boolean flag to run the encumbrance command before fetching the value.
